src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_f16_depthwise.cl

```diff
@@ -261,6 +261,34 @@ KERNEL(convolution_depthwise)(
             }
         }
     }
+#else // X_BLOCK_SIZE == 1
+    OUTPUT_TYPE res;
+#if OUTPUT_LEFTOVERS
+    if ((f_block + 1) * FEATURE_SLICE_SIZE >= OUTPUT_FEATURE_NUM)
+    {
+#if HAS_FUSED_OPS
+        uint i = 0;
+        FUSED_OPS_SCALAR;
+        res = FUSED_OPS_RESULT_SCALAR;
+#else
+        res = TO_OUTPUT_TYPE(dst[0]);
+#endif // HAS_FUSED_OPS
+        if (x < OUTPUT_SIZE_X && f_block * FEATURE_SLICE_SIZE + lid < OUTPUT_FEATURE_NUM)
+            output[output_offset + x * output_x_pitch + lid] = res;
+    }
+    else
+#endif // OUTPUT_LEFTOVERS
+    {
+#if HAS_FUSED_OPS
+        uint i = 0;
+        FUSED_OPS_SCALAR;
+        res = FUSED_OPS_RESULT_SCALAR;
+#else
+        res = TO_OUTPUT_TYPE(dst[0]);
+#endif // HAS_FUSED_OPS
+        output[output_offset + x * output_x_pitch + lid] = res;
```

Comment:
But  I think the BLOCK_WRITE can be used here. Could you try 
`OUTPUT_BLOCK_WRITE(output, output_offset + x * output_x_pitch, res);` and check the accuracy? Thanks!

Reply:
Fixed and check the accuracy is same as before. Thanks